### ISSUE TITLE
Create a Nuke Op Drop Ship B: 25

### ISSUE THREAD

**Issue by DooblyNoobly** [unitystation/unitystation#974]
# TODO:
Create the Nuke Op Drop Ship on its own separate matrix and park it a short distance away from the OutPost station on OutPostDM and the parked TrailBlazer on FlashLightDM.

## Criteria:
1. Create a new matrix for the DropShip
2. Create the Tilemaps for the DropShip and set them up correctly
3. Place a Shuttle Control console in the bridge of the DropShip (depends on #971)
4. Add weapons and armor for the players who spawn there
5. Add the Spawn point for the Syndicate role and place on the DropShip
6. Test to make sure the spawn works and the shuttle control works correctly on a headless + 2 client environment

Bounty of 25usd will be paid to the contributor on successful merge

**Comment by dapocalypse** [comment]
Can I do this as my first one. Please no one steal it from me

**Comment by DooblyNoobly** [comment]
Criteria 3 and 6 could not be met as they are dependent on other TODO's
